core/deployment/src/main/java/io/quarkus/deployment/recording/BytecodeRecorderImpl.java

```diff
@@ -255,7 +255,8 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
                         return "Runtime proxy of " + returnType + " with id " + key;
                     }
                     throw new RuntimeException(
-                            "You cannot invoke directly on an object returned from the bytecode recorded, you can only pass is back into the recorder as a parameter");
+                            "You cannot invoke " + method.getName()
+                                    + "() directly on an object returned from the bytecode recorded, you can only pass is back into the recorder as a parameter");
```

Comment:
They are preexisting but:
- I'm wondering if `from the bytecode recorded` should instead be `from the bytecode recorder`?
- s/pass is back/pass it back/

Reply:
Good catch.